src/index.html

```diff
@@ -17,11 +17,439 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is pointing to a `.scss` file. Ensure that this file is being preprocessed into CSS, as browsers cannot directly interpret SASS files. If not, change the link to point to the compiled CSS file.